runtime/compiler/runtime/ClassUnloadAssumption.cpp

```diff
@@ -402,19 +401,17 @@ void TR_RuntimeAssumptionTable::markForDetachFromRAT(OMR::RuntimeAssumption *ass
    assumption->markForDetach();
    _marked++;
    }
-
 /**
  * Traverse the entire RAT detaching and reclaiming all marked assumptions.
- * This assumes that the assumptions have already been detached from the
+ * This assumes that the assumptions have already been detached from the 
```

Comment:
Please remove the blank at the end of this line.  It is causing the "Line Endings Check" error that is being reported on this file.